src/common/CommonUtils.ts

```diff
@@ -215,51 +215,99 @@ export class CommonUtils {
     }
 
     /**
-     * Execute a command asynchronously.
+     * Execute a command asynchronously using child_process.exec()
      *
      * @param command The command to be executed.
      * @returns A promise containing the results of stdout and stderr
      */
     public static async executeCommandAsync(
-        command: string,
-        forwardStdOut = false,
-        forwardStdErr = false
+        command: string
     ): Promise<{ stdout: string; stderr: string }> {
         return new Promise<{ stdout: string; stderr: string }>(
             (resolve, reject) => {
-                CommonUtils.logger.debug(`Executing command: '${command}'.`);
-                const prc = childProcess.exec(
-                    command,
-                    (error, stdout, stderr) => {
-                        if (error) {
-                            CommonUtils.logger.error(
-                                `Error executing command '${command}':`
-                            );
-
-                            // also include stderr & stdout for more detailed error
-                            let msg = error.message;
-                            if (stderr && stderr.length > 0) {
-                                msg = `${msg}\nstderr:\n${stderr}`;
-                            }
-                            if (stdout && stdout.length > 0) {
-                                msg = `${msg}\nstdout:\n${stdout}`;
-                            }
-
-                            CommonUtils.logger.error(msg);
-                            reject(error);
-                        } else {
-                            resolve({ stdout, stderr });
```

Comment:
Reverting these back to the way it was before. These were added in order to be used by our WDIO-related commands in `lwc-dev-mobile`, which now will use `spawnCommandAsync` instead.